nextjs-13/src/components/PublicLayout/index.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { render, screen } from '@testing-library/react';
+import { useSession } from 'next-auth/react';
+
+import PublicLayout from './';
+
+jest.mock('next-auth/react');
+jest.mock('next/navigation');
+
+describe('PublicLayout', () => {
+  it('renders the component', () => {
+    useSession.mockReturnValue({ status: 'authenticated' });
+
+    render(
+      <PublicLayout>
+        <></>
+      </PublicLayout>
+    );
+
+    expect(screen.getByTestId('public-newsletter')).toBeInTheDocument();
+  });
+
+  describe('Session status is "unauthenticated', () => {
+    it('renders the banner', () => {
+      useSession.mockReturnValue({ status: 'unauthenticated' });
+
+      render(
+        <PublicLayout>
+          <></>
+        </PublicLayout>
+      );
+
+      expect(screen.getByTestId('banner')).toBeInTheDocument();
+    });
+  });
+
+  describe('Session status is not "unauthenticated', () => {
```

Comment:
Double negation which is a bit hard to understand:

```suggestion
  describe('Session status is "authenticated', () => {
```

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/127/commits/3e6348d4b8db77c4a4fd623f7f6aad17dd84f701)